ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/TransactionEIP1559Test.java

```diff
@@ -39,22 +40,24 @@ public void buildEip1559Transaction() {
         List.of(
             new AccessListEntry(
                 Address.fromHexString("0x000000000000000000000000000000000000aaaa"),
-                List.of(Bytes32.ZERO)));
+                List.of(
+                    Bytes32.fromHexString("0x000000000000000000000000000000000000aaaa"),
+                    Bytes32.fromHexString("0x34Ee1552077D197D511b3C615459b8E597d9eE04"))));
     final Transaction tx =
         Transaction.builder()
-            .chainId(new BigInteger("1559", 10))
+            .chainId(new BigInteger("1981", 10))
             .nonce(0)
             .value(Wei.ZERO)
             .gasLimit(30000)
-            .maxPriorityFeePerGas(Wei.of(2))
+            //            .maxPriorityFeePerGas(Wei.of(2))
```

Comment:
Any reason to keep these commented out lines in here?

Reply:
fixed!